### DISCUSSION TITLE
Scene State Decision

### DISCUSSION THREAD

**Discussion by ShadowcasterIT** [hugobloem/stateful_scenes#98] [Q&A] [answered]
Hello,

Is there a way to change how a scene decides whether it is on/off using the attributes you support?

For example, I have a scene with multiple lights. Most of the lights are Govee with an effect applied, but one of the lights is Tuya and just has a static color applied. State detection for this scene does not seem to function properly.  I assume it could be because the RGB attribute is constantly changing for the Govee lights. I would like to be able to edit the decision maker to only look at the effect attribute for the Govee lights but only look at the RGB attribute for the Tuya light.

Is this possible? Or is there something else I am missing that should make this scene state work properly as is?

**Comment by hugobloem** [comment, marked as answer]
Hi, 

You can edit your `scene.yaml` file to remove the rgb attributes and only keep the effect attribute. If you edit the scene via the UI it automatically stores all possible attributes, hence you want to change it in the yaml file.
